src/components/AsideSection/AsideSection.css

```diff
@@ -3,6 +3,12 @@
 @import "src/css/variables/gutters.css";
 
 .y-aside-section--header {
-  border-bottom: 1px solid $borderColor;
-  padding-bottom: $gutterWidth__xSmall;
+  border-bottom: 1px solid $borderColor__secondary;
+  padding-bottom: $gutterWidth__xxSmall;
+  white-space: pre;
```

Comment:
what happens when there is a very long title, does it just overflow to the right?

can you explain the problem we're trying to solve? if we just want to keep the `action` text from wrapping then we should probably add a "nowrap" option to FixedGridColumn, we've needed that in other places too

Reply:
I tried adding white-space: nowrap; to all FixedGridColumn, it doesn't wrap but looks like: 
<img width="261" alt="screen shot 2018-02-21 at 6 13 09 pm" src="https://user-images.githubusercontent.com/31256966/36516466-f4917d32-1732-11e8-91b0-313f47523564.png">

My changes allow text to grow until it meets another column, and only then jump to the next line
<img width="253" alt="screen shot 2018-02-21 at 6 14 12 pm" src="https://user-images.githubusercontent.com/31256966/36516487-0aa7b866-1733-11e8-8932-6aae524fed3d.png">

Reply:
cool thanks. let's ask Design how this should behave

Reply:
I confirmed all the behavior with @adrianmg and showed him what it looks like now after changes :)